addons/website_sale/controllers/main.py

```diff
@@ -997,3 +997,29 @@ def country_infos(self, country, mode, **kw):
             states=[(st.id, st.name, st.code) for st in country.get_website_sale_states(mode=mode)],
             phone_code=country.phone_code
         )
+
+    @http.route("/shop/cart/<token>", type='http', auth="public", website=True)
+    def access_abandoned_cart(self, token=None, **post):
+        """
+        Token will be pass from revival email template and it will redirect to the selected cart.
+        """
+        order = request.env['sale.order'].sudo().search([('access_token', '=', token)], limit=1)
```

Comment:
You are redirected to your current cart content:
[cart route is '/shop/cart/'](https://github.com/odoo-dev/odoo/blob/2f791e81927ad9f9eed1856554cc5404af08d8b9/addons/website_sale/controllers/main.py#L337)